packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.css

```diff
@@ -1,24 +1,57 @@
+:host {
+  --x-translation: var(--adapted-x-translation, 0px);
+  --max-width: var(--adapted-max-width, 20rem);
+}
+
 .tooltip__content {
   left: 50%;
-  transform: translate(-50%, -100%);
+  transition: transform 0.05s linear 0.05s, opacity 0.1s linear 0s, visibility 0.1s linear 0s;
+  z-index: 2;
+}
+
+.tooltip__content--positioned-above {
+  transform: translate(calc(-50% + var(--x-translation)), -100%);
```

Comment:
Can we split the translate in to translateX and translateY just for clarity?

You could have one single class 

```
.tooltip__content {
  transform: translateX(calc(-50% + var(--x-translation)));
}

.tooltip__content--positioned-above {
  transform: translateY(-100%);
}

.tooltip__content--positioned-below {
  transform: translateY(0.7rem);
}
```

Not sure that works, but it feels more readable to me

Reply:
Unfortunately this will not work cause the transform attributes will be overwritten between these three classes, 
True that each class modify a given axis but they all are under the CSS attribute `transform` so for example  if you add `tooltip__content--positioned-above` you will lose the translateX of `tooltip__content`.

Reply:
gotcha yea I wasn't sure, all good then